src/main/scala/com/databricks/labs/overwatch/pipeline/GoldTransforms.scala

```diff
@@ -493,6 +493,82 @@ trait GoldTransforms extends SparkSessionWrapper {
 
   }
 
+
+  protected def buildNotebookCommands(
+                                   notebook: PipelineTable,
+                                   clsfIncrementalDF : DataFrame,
+                                 )(auditIncrementalDF: DataFrame): DataFrame = {
+
+
+    val notebookLookupTSDF = notebook.asDF
+      .select("organization_id", "notebook_id", "notebook_path", "notebook_name", "unixTimeMS","date")
+      .withColumnRenamed("notebook_id", "notebookId")
+      .filter('notebookId.isNotNull)
+      .distinct
+      .toTSDF("unixTimeMS", "organization_id","date","notebookId")
+
+    val clsfLookupTSDF = clsfIncrementalDF
+      .select(
+        "organization_id", "state_start_date", "unixTimeMS_state_start", "cluster_id", "cluster_name",
+        "custom_tags", "node_type_id", "current_num_workers", "uptime_in_state_H", "total_cost")
+      .distinct
+      .withColumnRenamed("unixTimeMS_state_start", "unixTimeMS")
+      .withColumnRenamed("cluster_id", "clusterId")
+      .withColumnRenamed("current_num_workers", "node_count")
+      .withColumn("totalCostPMS", col("total_cost") / col("uptime_in_state_H")/lit(3600))
```

Comment:
total_cost shoudl be total_dbu_cost since you're creating field `estimated_dbu_cost`. total_cost includes compute costs which we don't want for this.